scriptler/addGitCleanCheckout.groovy

```diff
@@ -0,0 +1,36 @@
+/*** BEGIN META {
+  "name" : "add git clean before checkout",
+  "comment" : "This script will add git clean after checkout to all Jenkins items ",
+  "parameters" : [],
+  "authors" : [
+    { name : "Zamir Ivry - zamir.ivry@gmail.com" }
+  ]
+} END META**/
+
+for (item in Hudson.instance.items) {
+    println item.getSCMs();
+    for(scm in item.getSCMs())
+    {
```

Comment:
Could you clean up the spacing and curly braces? i.e., space after "if" or "for", opening curly braces on the same line, get rid of the tab between "instanceof" and "hudson.plugins.git.GitSCM". Thanks.